src/EditorFeatures/Core.Cocoa/Microsoft.CodeAnalysis.EditorFeatures.Cocoa.csproj

```diff
@@ -43,11 +43,6 @@
     <PackageReference Include="System.ValueTuple" Version="$(SystemValueTupleVersion)" />
   </ItemGroup>
 
-  <ItemGroup>
-    <!-- Manual exclusions for unwanted interop assemblies -->
-    <PackageReference Include="Microsoft.VisualStudio.Interop" Version="$(MicrosoftVisualStudioInteropVersion)" ExcludeAssets="all" PrivateAssets="all" />
```

Comment:
FYI @davidwengier to confirm this is the right change

Reply:
Honestly, I can't remember why this was added... but if its being removed from everywhere it should be fine. If its somehow used transitively from some other project, we'll find out on the next VS Mac insertion, and can go from there.

How the next VS Mac insertion will work with Dev17 bits I don't know anyway 😀

Reply:
I believe this was added to make sure we didn't have dependency to old interops: at that time, we did't have clean, merged dev17 interop packages available yet because of all the inter-dependencies. Now all the packages are updated so there's no need for this workaround anymore. At least this was the reason for other projects, I just don't know things well enough to decide if this was the only reason for you Mac folks :)